<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogViewForm</class>
 <widget class="QWidget" name="LogViewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QWidget" name="searchToolWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="seachLineEdit"/>
      </item>
      <item>
       <widget class="QCheckBox" name="caseSensCheckBox">
        <property name="text">
         <string>Case sensivity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="backCheckBox">
        <property name="text">
         <string>Seach backward</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchPushButton">
        <property name="toolTip">
         <string>Search</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="infoviewer.qrc">
          <normaloff>:/InfoViewer/images/search.png</normaloff>:/InfoViewer/images/search.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="searchAllPushButton">
        <property name="toolTip">
         <string>Search all</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="infoviewer.qrc">
          <normaloff>:/InfoViewer/images/searchAll.png</normaloff>:/InfoViewer/images/searchAll.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Hide panel</string>
        </property>
        <property name="text">
         <string>X</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>false</bool>
     </property>
     <widget class="CustomTextViewWidget" name="logViewWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>720</width>
        <height>497</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,1,0">
     <item>
      <widget class="QCheckBox" name="m_debugCheckBox">
       <property name="text">
        <string>Debug</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_infoCheckBox">
       <property name="text">
        <string>Information</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_warnCheckBox">
       <property name="text">
        <string>Warning</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="m_errorCheckBox">
       <property name="text">
        <string>Error</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_filterLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_applyFilterPushButton">
       <property name="toolTip">
        <string>Apply filter</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:/MainWindow/images/apply.png</normaloff>:/MainWindow/images/apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTextViewWidget</class>
   <extends>QWidget</extends>
   <header>CustomTextViewWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="infoviewer.qrc"/>
 </resources>
 <connections/>
</ui>
